articles.tex

```diff
@@ -99,7 +99,9 @@
 \asubsection{Semantic Changes}
 Any semantic change to the Constitution requires the change to be proposed in writing for discussion at a House Meeting.
 Any modifications made due to the discussion are added to the written proposal and the modified proposal is posted in the House during the week.
-The final proposal is presented the following week and ballots are distributed for a ballot House vote as described in \ref{Balloted Vote}.
+The final proposal is presented the following week at the end of House Meeting.
```

Comment:
I would leave out the "end of" line. The chair of the meeting will figure out what time in the meeting makes sense.